roles/cloud-bootstrap/tasks/network.yml

```diff
@@ -0,0 +1,66 @@
+---
+- name: Create control-plane network
+  os_network:
+    cloud: "{{ cloud }}"
+    name: control-plane
```

Comment:
So in all of these os_* tasks, we need to specify the project we want to create them in with an auth: parameter, ie:
auth:
  project_name: bonnyci